client/me/purchases/manage-purchase/index.jsx

```diff
@@ -524,7 +524,7 @@ class ManagePurchase extends Component {
 		};
 
 		return (
-			<CompactCard href={ link } onClick={ onClick }>
+			<CompactCard href={ link } className={ `remove-purchase__card` } onClick={ onClick }>
```

Comment:
Minor, but you can write this with less JavaScript as:

```suggestion
			<CompactCard href={ link } className="remove-purchase__card" onClick={ onClick }>
```

(We really only need backticks when doing string interpolation or multi-line strings, and we also don't need to drop back to JS with the curly braces unless we need to do something in JS.)